src/TabButton/tabButton.css

```diff
@@ -3,36 +3,50 @@
 
 .default
 {
-    margin-bottom:	0;
-    color:          var( --PC-BLUE ) ;
+    margin:             0 0 -1px 0;
+    color:              var( --PC-GREY--L70 ) ;
 
-    box-shadow:     none;
+    border-radius:      0;
+
+    text-transform:     initial;
```

Comment:
As a general practice I think we should avoid `initial` or `unset` since we can’t be 100% sure what those values will be.

In this case I would explicitly set `text-transform: none`.